src/NClient.Providers/NClient.Providers.Serialization.Protobuf.ProtobufNet/Extensions/ProtobufSerializationExtensions.cs

```diff
@@ -0,0 +1,77 @@
+using NClient.Common.Helpers;
+
+namespace NClient.Providers.Serialization.Protobuf.ProtobufNet.Extensions
+{
+    public static class ProtobufSerializationExtensions
+    {
+        /// <summary>
+        /// Sets ProtobufNet based <see cref="ISerializerProvider"/> used to create instance of <see cref="ISerializer"/>.
+        /// </summary>
+        /// <param name="optionalBuilder"></param>
+        /// <typeparam name="TClient"></typeparam>
+        /// <typeparam name="TRequest"></typeparam>
+        /// <typeparam name="TResponse"></typeparam>
+        /// <returns></returns>
+        public static INClientOptionalBuilder<TClient, TClient, TResponse> WithProtobufNetSerialization<TClient, TRequest, TResponse>(
```

Comment:
The second parameter TClient must be replaced with the TRequest. This needs to be done in other methods too

Reply:
Fixed.